WordPress/src/main/java/org/wordpress/android/ui/posts/services/AztecImageLoader.java

```diff
@@ -60,6 +67,8 @@ public void onResponse(ImageLoader.ImageContainer response, boolean isImmediate)
                     // isImmediate is true as soon as the request starts.
                     callbacks.onImageFailed();
                 } else if (bitmap != null) {
+                    final String cacheKey = url + maxWidthForEditor;
+                    WordPress.getBitmapCache().putBitmap(cacheKey, bitmap);
```

Comment:
Any reason we cache image here? Current image loader already caching the image, and in case of the local file, we are reading it from local storage, which is fast.

Reply:
I wanted to avoid to invoke the network lib (and all of its code) for network pictures when possible. For local pictures ` byte[] bytes = ImageUtils.createThumbnailFromUri(
                   context, Uri.parse(url), maxWidthForEditor, null, orientation);` is not that fast. Reading it from the cache is way faster.

I can remove the cache if you think it's more like an "over-optimization".

Reply:
I was more concerned with caching the image in memory twice. I haven't really looked at how the caching is implemented, and maybe it's not a big deal? Anyway, if it improves performance I don't mind keeping it :)

Reply:
Our cache is just a LRU implementation that uses 4Mb of RAM, that gets evicted when the device runs low in memory.